docs/tutorial/02_deploying_magma_orchestrator.md

```diff
@@ -0,0 +1,225 @@
+  id: "2"
+  results:
+    fluentd: a13611fd67ae84df5bd40c4c7fe892d5-1909378219.us-east-2.elb.amazonaws.com
+    nginx-proxy: a7b65619cbf8443feb05910823b6c42b-748407990.us-east-2.elb.amazonaws.com
+    orc8r-bootstrap-nginx: ab4feaf807c9440b8b021395f0422b26-1385734990.us-east-2.elb.amazonaws.com
+    orc8r-clientcert-nginx: ac63855b699064f5b825e707cfe290f0-1144570992.us-east-2.elb.amazonaws.com
+    orc8r-nginx-proxy: a6ae77105db624f06bb37f18843db925-911888461.us-east-2.elb.amazonaws.com
+  status: completed
+  timing:
+    completed: 2023-03-08 19:26:46 +0000 UTC
+    enqueued: 2023-03-08 19:26:44 +0000 UTC
+    started: 2023-03-08 19:26:45 +0000 UTC
+```
+
+The hostnames associated to each service will differ from those shown here.
+
+Create a file named `dns.json` with the following content:
+
+```json title="dns.json"
+{
+  "Comment": "CREATE CNAME records",
+  "Changes": [
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "bootstrapper-controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ab4feaf807c9440b8b021395f0422b26-1385734990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "api.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a6ae77105db624f06bb37f18843db925-911888461.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "controller.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "ac63855b699064f5b825e707cfe290f0-1144570992.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    },
+    {
+      "Action": "CREATE",
+      "ResourceRecordSet": {
+        "Name": "*.nms.<your domain name>",
+        "Type": "CNAME",
+        "TTL": 300,
+        "ResourceRecords": [
+          {
+            "Value": "a7b65619cbf8443feb05910823b6c42b-748407990.us-east-2.elb.amazonaws.com"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+!!! note
+    Replace each resource record value with the ones received from the previous step using the following Service/Hostname scheme:
+
+    | Service                                | Hostname                              | 
+    |----------------------------------------|---------------------------------------|
+    | `<orc8r-bootstrap-nginx External IP>`  | `bootstrapper-controller.<your domain name>` | 
+    | `<orc8r-nginx-proxy External IP>`      | `api.<your domain name>`                     | 
+    | `<orc8r-clientcert-nginx External IP>` | `controller.<your domain name>`              | 
+    | `<nginx-proxy External IP>`            | `*.nms.<your domain name>`                   | 
```

Comment:
In this tutorial, we should replace IP with URL or FQDN, the output on AWS is not IPs.